src/k8s-extension/HISTORY.rst

```diff
@@ -6,6 +6,7 @@ Release History
 1.4.1
 ++++++++++++++++++
 * microsoft.azureml.kubernetes: Fix sslSecret parameter in update operation
+* microsoft.azuremonitor.containers.metrics : public preview support for managed prometheus in ARC clusters
 
```

Comment:
Create a new version 1.4.2 and add this comment

Reply:
Its okay to add it in the version '1.4.1' itself as the most recent version is 1.4.0 . The history was updated in another pr but was not released.

fyi @NarayanThiru, pr link --> [https://github.com/AzureArcForKubernetes/azure-cli-extensions/pull/224/files](https://github.com/AzureArcForKubernetes/azure-cli-extensions/pull/224/files)